dotnetUnoFrameBuffer/.doc/README.md

```diff
@@ -1,4 +1,6 @@
-# .NET 6 C# Uno Platform Frame Buffer template specific documentation
+# .NET 6 C# Uno Platform Fra
```

Comment:
I think here a part of the phrase has been removed: "# .NET 6 C# Uno Platform FrameBuffer template specific documentation"